### PR TITLE
home_screen, navigation, kinopoiskApi, mapperAnilibriaUrl: changed, added

### PR DESCRIPTION
**Pull request by Razor0744**

### CODE REVIEW

File: app/src/main/java/com/example/movopfy/features/home/presentation/viewmodel/HomeViewModel.kt [hunk 1]
```diff
@@ -2,29 +2,87 @@ package com.example.movopfy.features.home.presentation.viewmodel
...
+class HomeViewModel(
+    private val anilibriaRepository: AnilibriaRepository,
+    private val kinopoiskRepository: KinopoiskRepository
+) :
     ViewModel() {
 
     private val _uiState = MutableStateFlow<HomeUiState>(HomeUiState.Loading)
     val uiState: StateFlow<HomeUiState> = _uiState.asStateFlow()
 
+    private var anime = emptyList<WaitingListToday>()
```
Comment (MaximSemashko) [inline app/src/main/java/com/example/movopfy/features/home/presentation/viewmodel/HomeViewModel.kt]: Please rename it

File: app/src/main/java/com/example/movopfy/features/home/presentation/viewmodel/HomeViewModel.kt [hunk 2]
```diff
@@ -2,29 +2,87 @@ package com.example.movopfy.features.home.presentation.viewmodel
...
+    private var movieList = mutableListOf<Pair<String, List<KinopoiskDocs>>>()
+
     init {
         viewModelScope.launch {
-            val schedules = getWaitingListTodayUseCase.execute(day = 6)
+            anime =
+                mapAnilibriaUrlToImageUrl(anilibriaSchedule = anilibriaRepository.getSchedule()[6])
```
Comment (MaximSemashko) [inline app/src/main/java/com/example/movopfy/features/home/presentation/viewmodel/HomeViewModel.kt]: `6` -  is a magic number. Please move to a constant. https://en.wikipedia.org/wiki/Magic_number_(programming)#:~:text=In%20computer%20programming%2C%20a%20magic,see%20List%20of%20file%20signatures

File: app/src/main/java/com/example/movopfy/features/home/presentation/viewmodel/HomeViewModel.kt [hunk 3]
```diff
@@ -2,29 +2,87 @@ package com.example.movopfy.features.home.presentation.viewmodel
...
+
+
+            _uiState.emit(HomeUiState.Loaded(anime = anime, movieList = movieList))
+        }
+
+        viewModelScope.launch {
+            movieList.add(
+                Pair(
+                    "Ужасы",
```
Comment (MaximSemashko) [inline app/src/main/java/com/example/movopfy/features/home/presentation/viewmodel/HomeViewModel.kt]: "Ужасы, 1, +ужасы" etc - please move them into a constants

File: app/src/main/java/com/example/movopfy/uiComponents/navigation/BottomNavigationBar.kt [hunk 4]
```diff
@@ -24,7 +24,7 @@ fun BottomNavigationBar(
 ) {
     NavigationBar(
         containerColor = BackgroundNavBar,
-        modifier = Modifier.height(MaterialTheme.size.height)
+        modifier = Modifier.height(MaterialTheme.size.heightComponent)
```
Comment (MaximSemashko) [inline app/src/main/java/com/example/movopfy/uiComponents/navigation/BottomNavigationBar.kt]: strange name `heightComponent`

File: app/src/main/java/com/example/movopfy/features/home/presentation/viewmodel/HomeViewModel.kt (hunk 1 above)
Comment (Razor0744) [reply app/src/main/java/com/example/movopfy/features/home/presentation/viewmodel/HomeViewModel.kt]: did

File: app/src/main/java/com/example/movopfy/features/home/presentation/viewmodel/HomeViewModel.kt (hunk 2 above)
Comment (Razor0744) [reply app/src/main/java/com/example/movopfy/features/home/presentation/viewmodel/HomeViewModel.kt]: sorry, it is not a magic number, this number I will get from local date. This should be the current day

Comment (Razor0744) [reply app/src/main/java/com/example/movopfy/features/home/presentation/viewmodel/HomeViewModel.kt]: I forgot change it, this is my mistake

Comment (Razor0744) [reply app/src/main/java/com/example/movopfy/features/home/presentation/viewmodel/HomeViewModel.kt]: I did it

File: app/src/main/java/com/example/movopfy/features/home/presentation/viewmodel/HomeViewModel.kt (hunk 3 above)
Comment (Razor0744) [reply app/src/main/java/com/example/movopfy/features/home/presentation/viewmodel/HomeViewModel.kt]: I did

File: app/src/main/java/com/example/movopfy/uiComponents/navigation/BottomNavigationBar.kt (hunk 4 above)
Comment (Razor0744) [reply app/src/main/java/com/example/movopfy/uiComponents/navigation/BottomNavigationBar.kt]: and which name I should use? heightButton or just height?

Comment (Razor0744) [reply app/src/main/java/com/example/movopfy/uiComponents/navigation/BottomNavigationBar.kt:27]: I did

**Pull request merged**